src/mantine-core/src/components/Tooltip/Tooltip.test.tsx

```diff
@@ -80,11 +80,28 @@ describe('@mantine/core/Tooltip', () => {
     expect(screen.queryAllByText('test-tooltip')).toHaveLength(0);
   });
 
+  it('supports open delay', async () => {
+    jest.useFakeTimers();
+
+    await renderWithAct(
+      <Tooltip {...defaultProps} openDelay={500}>
+        <button type="button">test-target</button>
+      </Tooltip>
+    );
+    expect(screen.queryAllByText('test-tooltip')).toHaveLength(0);
+    await actAsync(() => userEvent.hover(screen.getByRole('button')));
+    expect(screen.queryAllByText('test-tooltip')).toHaveLength(0);
+    await actAsync(() => userEvent.unhover(screen.getByRole('button')));
+    expect(screen.queryAllByText('test-tooltip')).toHaveLength(0);
+    await actAsync(() => jest.runAllTimers());
+    expect(screen.queryAllByText('test-tooltip')).toHaveLength(0);
+  });
+
```

Comment:
This test does not check that tooltip was rendered with delay, instead it checks that it was never there, it's meaningless